game/scripts/npc/abilities/shadow_demon_disruption.txt

```diff
@@ -52,22 +52,22 @@
       "03"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "illusion_outgoing_damage"                        "-70.0 -55.0 -40.0 -25.0 -10.0 5.0"
+        "illusion_outgoing_damage"                        "-70.0 -50.0 -30.0 -10.0 10.0 30.0" //OAA
```

Comment:
SD's disruption falls off a cliff in terms of damage output compared to early to mid game. Illusions do not get many of the benefits from items that OAA carries use late game to exponentially increase their DPS. This change will hopefully address the late-game issue SD has. Nerfing damage taken a tad because it's redundant levels 1-4 and I don't want to make it such an annoyance to deal with at oaa levels. It's going to hurt if you don't kill them but they're not hard to kill should be the motto for this hero.

Reply:
Afaik, SD isn't supposed to do dmg in lategame. Also you are making lvl 3 the same as Dark Seer ult lvl 1 - I am not a fan. Revert vanilla levels and buff OAA levels. If you insist on buffing vanilla levels, buff first 3 levels but leave the lvl 4 the same as vanilla. So for example you can do: -55/-45/-35/-25/-5/25 or -70/-55/-40/-25/5/35.